woostarter/patterns/about-store-features-list-large.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Title: Large store features list
+ * Slug: woostarter/store-features-list-large
+ * Categories: text, about
+ * Keywords: about, text
+ * Description: A list section with store featured services.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Large store features list","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|30"} -->
+<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"var:preset|spacing|40"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+<div class="wp-block-group alignwide"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","verticalAlignment":"top"}} -->
+<div class="wp-block-group"><!-- wp:image {"id":475,"width":"30px","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full is-resized"><img src="https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/icon-check.webp" alt="Check icon." class="wp-image-475" style="width:30px"/></figure>
```

Comment:
This `src` attribute needs to be updated to `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/icon-check.webp` before merging.